datasets/finance.py

```diff
@@ -1,32 +0,0 @@
-from benchopt import BaseDataset
```

Comment:
this will break existing code.  If we agree on deleting it, there should be a deprecation warning for a while.

Reply:
Ok, I re-added this file